AUTHORING_GUIDE.md

```diff
@@ -281,216 +191,203 @@ Args:
-    * py27(sample='./appengine/flexible/storage')
-    * py27(sample='./storage/api')
-    * py27(sample='./storage/cloud-client')
-    * py35(sample='./appengine/flexible/storage')
-    * py35(sample='./storage/api')
-    * py35(sample='./storage/cloud-client')
-    * readmegen(sample='./storage/api')
-    * readmegen(sample='./storage/cloud-client')
-    * readmegen(sample='./storage/transfer_service')
-
-Now you can use nox to run a specific session, for example, if you want to lint
-the storage cloud-client samples:
-
-    $ nox -s "lint(sample='./storage/cloud-client')"
-
-### Test environment setup
-
-Because all the tests here are system tests, you'll need to have a Google
-Cloud project with billing enabled. Once you have this configured, you'll
-need to set environment variables for the tests to be able to use your project
-and its resources. See `testing/test-env.tmpl.sh` for a list of all environment
-variables used by all tests. Not every test needs all of these variables.
-
-#### Google Cloud Storage resources
-
-Certain samples require integration with Google Cloud Storage (GCS),
-most commonly for APIs that read files from GCS. To run the tests for
-these samples, configure your GCS bucket name via the `CLOUD_STORAGE_BUCKET`
+```console
+$ nox -k storage -l
+* gae(sample='./appengine/standard/storage/api-client')
+* gae(sample='./appengine/standard/storage/appengine-client')
+* lint(sample='./appengine/flexible/storage')
+* lint(sample='./appengine/standard/storage/api-client')
+* lint(sample='./appengine/standard/storage/appengine-client')
+* lint(sample='./storage/api')
+* lint(sample='./storage/cloud-client')
+* lint(sample='./storage/transfer_service')
+* py27(sample='./appengine/flexible/storage')
+* py27(sample='./storage/api')
+* py27(sample='./storage/cloud-client')
+* py35(sample='./appengine/flexible/storage')
+* py35(sample='./storage/api')
+* py35(sample='./storage/cloud-client')
+* readmegen(sample='./storage/api')
+* readmegen(sample='./storage/cloud-client')
+* readmegen(sample='./storage/transfer_service')
+```
+
+Now you can use nox to run a specific session, for example, if you want to
+lint the storage cloud-client samples:
+
+```console
+$ nox -s "lint(sample='./storage/cloud-client')"
+```
+
+## Test Environment Setup
+
+Because all tests are system tests that use live resources, running tests
+requires a Google Cloud project with billing enabled, as covered under
+[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+Once you have your project created and configured, you'll need to set environment
+variables to identify the project and resources to be used by tests. See
+[testing/test-env.tmpl.sh](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/testing/test-env.tmpl.sh)
+for a list of all environment variables used by all tests. Not every test
+needs all of these variables.
+
+## Google Cloud Storage Resources
```

Comment:
This section might need an update, but I'm curious to hear other opinions. I feel like each sample may require resources more than just GCS - and they tend to specify the variables you need to fill in. I'm not sure that the ./resources section is kept up to date. I could be wrong though - so let's hear what the rest of the owners think!

Reply:
+1 this needs to be generalized to infrastructure resources as a whole.